### PR TITLE
Adding ansible file samples for snappi-sonic

### PR DESCRIPTION
**Pull request by selldinesh**
### Description of PR

Summary: Adding ansible file samples for snappi-sonic group
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
Adding ansible file samples for snappi-sonic inorder to run snappi_tests cases on the go
#### How did you do it?
Tested on arista dut
#### How did you verify/test it?
This was tested locally in our setup, this includes the devices , links and credentials used for ixnetwork api server

### CODE REVIEW

**Comment by developfast** [comment]
@wangxin can these sample files be merged to master? These are example devices

**Comment by selldinesh** [comment]
Hi @developfast ,
Request to approve this PR

File: tests/snappi_tests/README.md
Comment (sdszhang) [inline tests/snappi_tests/README.md:1]: pls add the snappi package installation steps.

**Review by sdszhang — CHANGES_REQUESTED** [review]
Waiting on update of README for snappi package

**Comment by abdosi** [comment]
@selldinesh :please resolve conflict.

File: tests/snappi_tests/README.md
Comment (selldinesh) [reply tests/snappi_tests/README.md:1]: added the steps

**Comment by selldinesh** [comment]
reopened https://github.com/sonic-net/sonic-mgmt/pull/13245

**Pull request closed without merging**